feedback/api/permissions.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
+
+
+class IsOwnerFeedback(BasePermission):
+    message = "You must be the owner of the object to view."
+
+    def has_object_permission(self, request, view, obj):
+        if request.method in SAFE_METHODS:
+            if obj.user == None:
+                # if user is anonymous then read-only
+                return True
+
+            # Check permissions for read-only request
+            elif obj.user == request.user:
```

Comment:
no need for custom permissions just use IsAuthenticated

Reply:
why is this branch merged? this is not fixed

Reply:
Actually, with this method, I am not allowing any type of users to put or patch.
Is it not the correct way to do it?

Reply:
explore this http://www.django-rest-framework.org/api-guide/generic-views/#listcreateapiview

Also, return only that user specific data in retrieving

Reply:
hey made this changes and pushed the new branch with PR raised.
https://github.com/projectlumos/project-lumos-backend/pull/37